websocket.go

```diff
@@ -220,18 +223,18 @@ func (c *Client) Send(dst *network.ServerIdentity, path string, buf []byte) ([]b
 		if err != nil {
 			return nil, NewClientError(err)
 		}
-		c.si = dst
+		c.connections[dest] = conn
 	}
 	defer func() {
 		if !c.keep {
-			c.Close()
+			c.Close(dest)
```

Comment:
If we close all connections how should errors be handled? Just return the last error returned by the connection or `nil` if no error occurred. 
Also now we don't handle the error returned by `c.Close()`, shouldn't the client do something if an error is returned?